Riot/Modules/Application/AppDelegate.swift

```diff
@@ -124,4 +118,21 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func application(_ application: UIApplication, didReceiveRemoteNotification userInfo: [AnyHashable: Any], fetchCompletionHandler completionHandler: @escaping (UIBackgroundFetchResult) -> Void) {
         self.legacyAppDelegate.application(application, didReceiveRemoteNotification: userInfo, fetchCompletionHandler: completionHandler)
     }
+    
+    // MARK: - Private
+    
+    private func setupFlexDebugger() {
```

Comment:
We try to put the less thing as possible in the AppDelegate. The main entry point for dependency injection, navigation management and setup singletons is AppCoordinator.